src/containers/jurisdictions/JurisdictionLabel.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styled from 'styled-components';
+
+import { N2, PP4 } from '../../core/style/Colors';
+import {
+  MEDIA_QUERY_JUR_SM,
+  MEDIA_QUERY_JUR_MD,
+  MEDIA_QUERY_JUR_LG
+} from '../../core/style/Sizes';
+
+const LabelWrapper = styled.span`
+  display: flex;
+  flex-basis: 4%;
+  flex-direction: column;
+  font-size: 12px;
+  margin: 0 10px 20px 10px;
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_SM}px) {
+    flex-basis: 10%;
+    font-size: 14px;
+  }
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_MD}px) {
+    flex-basis: 20%;
+  }
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_LG}px) {
```

Comment:
Check your logic here. `props.x` and `props.y` are undefined.

![screen shot 2018-06-25 at 10 00 04 am](https://user-images.githubusercontent.com/440299/41864400-bc35340a-785e-11e8-956b-abe45c45c75c.png)

Reply:
Fixed